tests/lib/rules/no-implied-eval.js

```diff
@@ -168,90 +200,162 @@ ruleTester.run("no-implied-eval", rule, {
 		},
 		{
 			code: "global.setTimeout(function() { x = 1; }, 100);",
-			languageOptions: { sourceType: "commonjs" },
+			languageOptions: {
+				sourceType: "commonjs",
+				globals: globals.browser,
+			},
 		},
 		{
 			code: "global.setInterval(function() { x = 1; }, 100);",
-			languageOptions: { sourceType: "commonjs" },
+			languageOptions: {
+				sourceType: "commonjs",
+				globals: globals.browser,
+			},
 		},
 		{
 			code: "global.execScript(function() { x = 1; }, 100);",
-			languageOptions: { sourceType: "commonjs" },
+			languageOptions: {
+				sourceType: "commonjs",
+				globals: globals.browser,
+			},
 		},
 		{
 			code: "global.setTimeout(foo, 100);",
-			languageOptions: { sourceType: "commonjs" },
+			languageOptions: {
+				sourceType: "commonjs",
+				globals: globals.browser,
+			},
 		},
 		{
 			code: "global.setInterval(foo, 100);",
-			languageOptions: { sourceType: "commonjs" },
+			languageOptions: {
+				sourceType: "commonjs",
+				globals: globals.browser,
+			},
 		},
 		{
 			code: "global.execScript(foo, 100);",
-			languageOptions: { sourceType: "commonjs" },
+			languageOptions: {
+				sourceType: "commonjs",
+				globals: globals.browser,
+			},
 		},
 		{
 			code: "globalThis.setTimeout(foo, 100);",
-			languageOptions: { ecmaVersion: 2020 },
+			languageOptions: { ecmaVersion: 2020, globals: globals.browser },
 		},
 
 		// only checks on top-level statements or window.*
-		"foo.setTimeout('hi')",
+		{
+			code: "foo.setTimeout('hi')",
+			languageOptions: { globals: globals.browser },
+		},
 
 		// identifiers are fine
-		"setTimeout(foo, 10)",
-		"setInterval(1, 10)",
-		"execScript(2)",
+		{
+			code: "setTimeout(foo, 10)",
+			languageOptions: { globals: globals.browser },
+		},
+		{
+			code: "setInterval(1, 10)",
+			languageOptions: { globals: globals.browser },
+		},
+		{
+			code: "execScript(2)",
+			languageOptions: { globals: globals.browser },
+		},
 
 		// as are function expressions
-		"setTimeout(function() {}, 10)",
+		{
+			code: "setTimeout(function() {}, 10)",
+			languageOptions: { globals: globals.browser },
+		},
 
 		// setInterval
-		"foo.setInterval('hi')",
-		"setInterval(foo, 10)",
-		"setInterval(function() {}, 10)",
+		{
+			code: "foo.setInterval('hi')",
+			languageOptions: { globals: globals.browser },
+		},
+		{
+			code: "setInterval(foo, 10)",
+			languageOptions: { globals: globals.browser },
+		},
+		{
+			code: "setInterval(function() {}, 10)",
+			languageOptions: { globals: globals.browser },
+		},
 
 		// execScript
-		"foo.execScript('hi')",
-		"execScript(foo)",
-		"execScript(function() {})",
+		{
+			code: "foo.execScript('hi')",
+			languageOptions: { globals: globals.browser },
+		},
+		{
+			code: "execScript(foo)",
+			languageOptions: { globals: globals.browser },
```

Comment:
```suggestion
			languageOptions: { globals: { execScript: false } },
```

`execScript` is not in `globals.browser`.